app/controllers/api/v1/posts_controller.rb

```diff
@@ -0,0 +1,37 @@
+module Api
+  module V1
+    class PostsController < ApiController
+      before_action :authorize_request
+
+      def list_comments
+        user_id = params[:user_id]
+        post_id = params[:post_id]
+        if user_id && post_id
+          comments = Comment.where(author_id: user_id, post_id: post_id)
+          render json: { status: 'SUCCESS', message: 'Loaded comments successfully', data: comments },
+                 status: :ok
+        else
+          render json: { status: 'ERROR', message: 'User or post not found' }, status: :not_found
+        end
+      end
```

Comment:
- [ ] Implement the following endpoints too

1. Has API endpoint that returns all post in JSON format - **GET :** `/api/users/:user_id/posts` or `/api/v1/users/:user_id/posts`
1. Has API endpoint that returns all comments for a post - **GET :** `/users/:user_id/posts/:id/comments`

Reply:
hello @Emmyn5600 , thank you so much for the constructive feedback. unfortunately this was not part of the requirements and i tried to do what the requirements demanded since i am time bound.

Could you please review this so we are on the same page.

Thank you!